pkg/freeze/containerd/pause.go

```diff
@@ -35,51 +45,57 @@ func New() (*Containerd, error) {
 	}
 
 	return &Containerd{
-		conn: conn,
+		conn:       conn,
+		containerd: c,
 	}, nil
 }
 
-// Freeze freezes the user container via the freezer cgroup.
-func (f *Containerd) Freeze(ctx context.Context, podName, containerName string) error {
+// Freeze freezes the user container(s) via the freezer cgroup.
+func (f *Containerd) Freeze(ctx context.Context, podName string) error {
 	ctrd, err := containerd.NewWithConn(f.conn)
```

Comment:
I suspect this will all be a bit neater/cleaner if the containerd.NewWithConn happens in the construction of the CRI (outside of Containerd) and we just have a f.containerd which is already dialed (that would let us drop the error from New, for a start, and merge these two fields in to one), but we can play with that in a follow-up, this all looks good 👍